lib/gs_schemas/capnp/network.capnp

```diff
@@ -76,6 +76,8 @@ interface AuthenticatedServerConnection @0xcc65c2f3643e6ae0 {
     bootstrapGameData @0 () -> (data: GameTypes.GameBootstrapData);
     # Sends a chat message to the server.
     sendChatMessage @1 (text: Text) -> ();
+    # Sends a throw action to the server.
+    sendThrowAction @2 (position: GameTypes.PositionData, throw: GameTypes.ThrowAction) -> ();
```

Comment:
I'd prefer `placeBlock` and `breakBlock` actions with the argument of `(tick: UInt64, blockPos: IVec3)`. Tick will contain the game tick counter for the action, which will be important for server-side validation.

And instead of `sendXAction`, just `x` should work